sw/device/tests/BUILD

```diff
@@ -915,12 +915,11 @@ opentitan_test(
 opentitan_test(
     name = "csrng_edn_concurrency_test",
     srcs = ["csrng_edn_concurrency_test.c"],
-    broken = cw310_params(tags = ["broken"]),
     exec_env = dicts.add(
         EARLGREY_TEST_ENVS,
         {
             "//hw/top_earlgrey:fpga_cw310_sival": None,
-            "//hw/top_earlgrey:fpga_cw310_sival_rom_ext": "broken",
+            "//hw/top_earlgrey:fpga_cw310_sival_rom_ext": "None",
```

Comment:
In this case you can remove the whole line since the default version is included in `EARLGREY_TEST_ENVS`.

Also, if this is working on the FPGAs we should enable the test on silicon too.

Suggestion:

```python
exec_env = dicts.add(
    EARLGREY_TEST_ENVS,
    EARLGERY_SILICON_OWNER_ROM_EXT_ENVS,
    {
        "//hw/top_earlgrey:fpga_cw310_sival": None,
        "//hw/top_earlgrey:silicon_creator": None,
    },
),
```